common/tools/dev-tool/src/util/testUtils.ts

```diff
@@ -23,15 +23,16 @@ async function shouldRunProxyTool(): Promise<boolean> {
 }
 
 export async function runTestsWithProxyTool(
-  testCommandObj: concurrently.CommandObj
+  testCommandObj: Partial<concurrently.Command>
 ): Promise<boolean> {
+  let result: Promise<concurrently.CloseEvent[]>;
   if (
```

Comment:
I'd suggest refactoring this section a little bit to use the if-statement only to determine the list of arguments passed to concurrently and running/awaiting the command once:

```typescript
const commands: concurrently.Command[] = [];
if (await shouldRunProxyTool()) {
  commands.push(/* true condition command */);
} else {
  commands.push(/* false condition command */);
}

await concurrently(commands).result;
```

Reply:
Not really easy to do in this case because the `if` statement case adds `ConcurrencyOptions` that is not on the `else` statement.

Reply:
Ah I see the extra args now. In that case LGTM. I still think it would be nice to have only a single call and to avoid this destructuring, but it's a minor nitpick.

Reply:
Changed to suggested code style